changelog/fix-docs-links-part-2

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix outdated documentation links.
```

Comment:
1, Clarify this a bit to distinguish it from our dev docs.
2, "Update" would be more correct, rather than "fix". We're not fixing anything here!?  

```suggestion
Type: update

Update outdated public documentation links on WooCommerce.com
```